pkgs/applications/audio/dgedit/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv
+, fetchgit
+, pkg-config
+, autoreconfHook
+, qt4
+, libao
+, libsndfile
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dgedit";
+  version = "0.10.0";
+
+  src = fetchgit {
+    url = "git://git.drumgizmo.org/dgedit.git/";
+    rev = "v${version}";
+    sha256 = "11mcfa6h61q0gk0ynfpqql4lnih0m3y2z9mdjq4p1219hrapxs5q";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ pkg-config autoreconfHook ];
+  buildInputs = [ qt4 libao libsndfile ];
+
+  meta = with stdenv.lib; {
+    description = "the DrumGizmo drumkit editor";
```

nitpick, should have caught this last time
```suggestion
    description = "The DrumGizmo drumkit editor";
```

---

Did you keep the small `the` on purpose?